### PR TITLE
zopfli: bump to 1.0.3

### PR DESCRIPTION
**Pull request by Ra-V2**

### CODE REVIEW

**Comment by korli** [comment]
Could you switch to cmake? I see mentions of Haiku in CMakeLists.txt, it could compile out of the box, and provide zopflipng

File: app-arch/zopfli/zopfli-1.0.3.recipe [hunk 1]
```diff
@@ -31,17 +32,23 @@ BUILD_REQUIRES="
...
+		-DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_INSTALL_PREFIX="$commandBinDir"
 	make $jobArgs
 }
 
 INSTALL()
 {
 	install -d "$commandBinDir"
 	install -t "$commandBinDir" zopfli
+	install -t "$commandBinDir" zopflipng
```
Comment (Begasus) [inline app-arch/zopfli/zopfli-1.0.3.recipe]: Didn't check, but isn't there a "make install"?

Comment (Ra-V2) [reply app-arch/zopfli/zopfli-1.0.3.recipe]: @Begasus When I try to do "make install" it says there is no rule to make target "install".

Comment (Begasus) [reply app-arch/zopfli/zopfli-1.0.3.recipe]: You probably want to change directory for that first :)
```diff
+	cd build
+	make install
```

Comment (Ra-V2) [reply app-arch/zopfli/zopfli-1.0.3.recipe]: Ok, nevermind. Now should be good I think.

**Comment by korli** [comment]
$prefix/include should be moved to $includeDir

**Pull request merged**